models/db/score.model.js

```diff
@@ -0,0 +1,16 @@
+'use strict'
+
+module.exports = (sequelize, DataTypes) => {
+  const Score = sequelize.define('Score', {
+    uri: DataTypes.STRING,
```

Comment:
You could add a custom setter for uri that downcases the uri before storing, if we can't set collation to case insensitive.
http://docs.sequelizejs.com/manual/tutorial/models-definition.html#getters-setters